docs/man/docker.1.md

```diff
@@ -90,7 +90,7 @@ unix://[/path/to/socket] to use.
   Set key=value labels to the daemon (displayed in `docker info`)
 
 **--log-driver**="*json-file*|*syslog*|*none*"
```

Comment:
Format of cli.md (the actual cmd output) and this manpage doesnt match. They follow different formats for documenting this as far as I can tell.